terraform/team-members-datadog/.terraform.lock.hcl

```diff
@@ -0,0 +1,24 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Shouldn't you ignore this file, though?

Reply:
No, we want to make sure that we're all using the same versions. `terraform init` fetches the versions that are locked in `.terraform.lock.hcl`, and only upgrades them when explicitly called with `terraform init -upgrade`.